src/server/pfs/server/server_test.go

```diff
@@ -2631,43 +2631,6 @@ func TestPutFileSplit(t *testing.T) {
 	}
 }
 
-func TestPutFileSplitDelete(t *testing.T) {
-	if testing.Short() {
-		t.Skip("Skipping integration tests in short mode")
-	}
-
-	c := getClient(t)
-	// create repos
-	repo := tu.UniqueString("TestPutFileSplitDelete")
-	require.NoError(t, c.CreateRepo(repo))
-	commit, err := c.StartCommit(repo, "master")
-	require.NoError(t, err)
-	_, err = c.PutFileSplit(repo, commit.ID, "line", pfs.Delimiter_LINE, 0, 0, false, strings.NewReader("foo\nbar\nbuz\n"))
-	require.NoError(t, err)
-	require.NoError(t, c.DeleteFile(repo, commit.ID, fmt.Sprintf("line/%016x", 0)))
-	_, err = c.PutFileSplit(repo, commit.ID, "line", pfs.Delimiter_LINE, 0, 0, false, strings.NewReader("foo\nbar\nbuz\n"))
-	require.NoError(t, err)
-	require.NoError(t, c.DeleteFile(repo, commit.ID, fmt.Sprintf("line/%016x", 5)))
-	_, err = c.PutFileSplit(repo, commit.ID, "line", pfs.Delimiter_LINE, 0, 0, false, strings.NewReader("foo\nbar\nbuz\n"))
-	require.NoError(t, err)
-
-	files, err := c.ListFile(repo, commit.ID, "line")
-	require.NoError(t, err)
-	require.Equal(t, 7, len(files))
-	for _, fileInfo := range files {
-		require.Equal(t, uint64(4), fileInfo.SizeBytes)
-	}
-
-	require.NoError(t, c.FinishCommit(repo, commit.ID))
-
-	files, err = c.ListFile(repo, commit.ID, "line")
-	require.NoError(t, err)
-	require.Equal(t, 7, len(files))
-	for _, fileInfo := range files {
-		require.Equal(t, uint64(4), fileInfo.SizeBytes)
-	}
-}
-
```

Comment:
Is this test no longer valid?

Reply:
This is testing the case where a user does a put file with a split then deletes files in the split. We discussed that this should not be allowed and instead the user should do a split using the atomic put file you are working on. This test breaks if we apply changes in mod revision order because of the way split file records are stored in etcd. The two deletes would get applied before the put file split in this case.

Reply:
Makes sense, could we replace it with a test that makes sure that we error when people try to do `put-file`s on open commits?